opentelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 
 ## vNext
 
+Added the ability to prevent recursive telemetry generation through new
+context-based suppression mechanisms. This feature helps prevent feedback loops
+and excessive telemetry when OpenTelemetry components perform their own
+operations.
```

Comment:
is this adding support for suppressing internal telemetry, or also the telemetry generated by dependencies which are instrumented using otel api.

Reply:
all. To be precise, as long as 
1. Otel context is carried forward, 
2. and suppression flag not over-ridden. (`test_nested_suppression_scopes` test shows when suppression flows/not-flows)